### DISCUSSION TITLE
How can I access changeLocale in my Nuxt app?

### DISCUSSION THREAD

**Discussion by serclino** [formkit/formkit#1297] [General]
Hello, how can I access `changeLocale` in my Nuxt app?
I have custom wrapper component around `<FromKit type="form" />` and inside that component I'd like to change locale for my FormKit's inputs (inside `onMounted`.
But apparently I'm not able to get to this function. Docs says to `import { changeLocale } from '@formkit/vue'`, which is not possible since I'm using Nuxt.
Any help please?

**Comment by justin-schroeder** [comment]
You can still import from `@formkit/vue`. If you are using pnpm without shameful hoisting, then just install @formkit/vue

**Reply by serclino** [reply]
I use yarn and after installation my IDE still yells at me that @formkit/vue doesn't contain `changeLocale`. **But it works so everything is fine.** Thank you.

P.S. Just for my curiosity -> Previously I was using:

```
 onMounted(() => {
   const formConfig: any = inject(Symbol.for('FormKitConfig'))
   formConfig.locale = language
 })
```

and it also worked. But when I tried to repeat this approach in another wrapper (for UI purposes), it crashed my app telling me it can't find locale. Do u know why?
